examples/md-flexible/CMakeLists.txt

```diff
@@ -4,29 +4,31 @@ file(
         "src/*.cpp"
         "src/*.h"
 )
+add_executable(md-flexible ${MDFlex_SRC})
+
+target_include_directories(md-flexible PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})
 
-if (AUTOPAS_INTERNODE_TUNING)
+include(autopas_yaml-cpp)
+
+option(AUTOPAS_INCLUDE_MPI "Enable this variable to use MPI." OFF)
+
+if (AUTOPAS_INTERNODE_TUNING OR AUTOPAS_INCLUDE_MPI)
     message(STATUS "AUTOPAS_INTERNODE_TUNING set, searching for mpi to link against md-flexible ...")
     find_package(MPI)
 
     if (NOT ${MPI_CXX_FOUND})
         message(FATAL_ERROR "cxx mpi could not be found, even though AUTOPAS_INTERNODE_TUNING was set.")
     else ()
         message(STATUS "cxx mpi found: ${MPI_CXX_COMPILER}")
+        set(MD_FLEXIBLE_USE_MPI ON)
+        add_definitions(-DAUTOPAS_INTERNODE_TUNING)
+        add_definitions(-DAUTOPAS_INCLUDE_MPI)
```

Comment:
AUTOPAS_INTERNODE_TUNING is still being used in WrapMPI.h
The goal is to replace it at most locations and only to keep it where actual internode communication is required.
The definition AUTOPAS_INCLUDE_MPI aims  to replace AUTOPAS_INTERNODE_TUNING at every other place.
Because I did not yet get to properly replace AUTOPAS_INTERNODE_TUNING we need both definitions here.

Reply:
Do you plan to do this in this PR? Otherwise add a TODO and maybe an issue

Reply:
I will do it in this pull request.